src/grid/submission/SubmissionGridBody.component.ts

```diff
@@ -22,9 +25,17 @@ export class SubmissionGridBodyComponent extends GridBodyComponent {
    */
   view(row: any, header: any) {
     const cellValue: any = get(row, header.key);
-    if (typeof header.component.getView === 'function') {
-      return header.component.getView(cellValue);
+    if (header.getView) {
+      return header.getView(cellValue);
+    } else {
+      if (header.component) {
+        if (typeof header.component.getView === 'function') {
+          return header.component.getView(cellValue);
+        }
+        return header.component.asString(cellValue);
+      } else {
+        return header.format ? formatDate(cellValue, header.format, '') : cellValue.toString();
```

Comment:
@vlad-shusterman  what is `header.format`? Is that just a string with date format like `'dd-mm-yyyy'` or something like that? If yes, I'm not sure I support introducing this additional field, there are several reasons for that:

1. This `format` field would only work for Date fields and for other types of columns it will not make any sense. Desire to format the field value to particular date format seems to me too specific for introducing it on the level of grid column. I'd prefer to use something more generic that would work for any type of column, see next point:

2. We already have generic `getView` method which could be used to achieve the same result (format value to particular date format), but this will be done on the level of the app that uses our angular-formio grid, not on the level of angular-formio grid itself (which for me seems semantically more correct)

So, could you please remove `format` property from header and column definitions and move date formatting logic to the app which uses angular-formio grid (by using `getView` method for the specific column for which you need to format the date)